gpbft/metrics.go

```diff
@@ -35,8 +35,11 @@ var (
 	}
 	attrSkipToRound  = attribute.String("to", "round")
 	attrSkipToDecide = attribute.String("to", "decide")
-	attrCacheHit     = attribute.String("status", "hit")
-	attrCacheMiss    = attribute.String("status", "miss")
+
+	attrCacheHit               = attribute.String("status", "hit")
+	attrCacheMiss              = attribute.String("status", "miss")
```

Comment:
nit: I'd use "cache" as the key here instead of status (e.g., in case we have other metrics that can have "status" in addition to cache hits/misses).